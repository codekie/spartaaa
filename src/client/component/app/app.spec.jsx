import React from 'react';
import { shallow } from 'enzyme';
import App from './app.jsx';

it('renders correctly', () => {
    expect(shallow(<App />)).toMatchSnapshot();
});
