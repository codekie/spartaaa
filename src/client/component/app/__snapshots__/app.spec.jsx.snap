// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Section
  className=""
  renderAs="section"
  size={null}
  style={Object {}}
>
  <Connect(Component) />
  <PanelBlock
    active={false}
    className=""
    renderAs="div"
  >
    <Control
      className=""
      fullwidth={false}
      iconLeft={false}
      iconRight={false}
      loading={false}
      renderAs="div"
      size={null}
      style={Object {}}
    >
      <Connect(Component) />
    </Control>
  </PanelBlock>
  <Connect(Component) />
  <Connect(Component) />
</Section>
`;
